### ISSUE TITLE
Would it be possible to add a grayscale print option?

### ISSUE THREAD

**Issue by Dima-369** [haji-ali/lp-transient#11]
`lp -o CNIJGrayScale=1` works for me on macOS, but not sure if this is printer dependent. I used `lpoptions -l` to find that option.

**Comment by haji-ali** [comment]
Yes, using `lpoptions` to get the printer-specific options is a planned feature (#2) but I haven't had time to look into it yet.

In the meantime, I just pushed a change that allows one to specify one extra option. I am hoping to change this so that it accepts multiple options, but I am still thinking about the correct interfaces.
